osgeo_importer/utils.py

```diff
@@ -92,6 +105,46 @@ def ensure_defaults(layer):
         layer.resource.catalog.save(fs)
 
 
+def create_vrt(file_path):
```

Comment:
I removed this in 46bf0db13f69e8f12d786b7d4b42fba7811328a3.  Any reason why its added back?